modules/acquia_cms_search/composer.json

```diff
@@ -8,8 +8,8 @@
         "drupal/acquia_cms_common": "2.0.x-dev",
         "drupal/acquia_search": "^3.1",
         "drupal/collapsiblock": "^4.0",
-        "drupal/facets": "2.0.5",
-        "drupal/facets_pretty_paths": "^1.3",
+        "drupal/facets": "^2.0",
```

Comment:
update to **"drupal/facets": "^2.0.6"**, OR **"drupal/facets": "^3.0"**, accordingly

Reply:
Done